scopes/ui-foundation/tree/drawer/drawer.module.scss

```diff
@@ -18,8 +18,9 @@
   user-select: none;
   font-weight: bold;
   padding: 0 8px;
-
+  line-height: 16px;
```

Comment:
consider changing to `1em`. so if we change the font size to 20px it will still look good